[2024-10-15T23:18:01.289+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:18:01.311+0000] {__init__.py:51} DEBUG - Loading core task runner: StandardTaskRunner
[2024-10-15T23:18:01.325+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:18:01.332+0000] {base_task_runner.py:68} DEBUG - Planning to run as the  user
[2024-10-15T23:18:01.335+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> from DB
[2024-10-15T23:18:01.359+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:18:01.370+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Task Instance Not Running' PASSED: True, Task is not in running state.
[2024-10-15T23:18:01.370+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Task Instance State' PASSED: True, Task state queued was valid.
[2024-10-15T23:18:01.371+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:18:01.371+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]>
[2024-10-15T23:18:01.372+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:18:01.378+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Task Concurrency' PASSED: True, Task concurrency is not set.
[2024-10-15T23:18:01.382+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Pool Slots Available' PASSED: True, There are enough open slots in default_pool to execute the task
[2024-10-15T23:18:01.382+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:18:01.383+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [queued]>
[2024-10-15T23:18:01.383+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-10-15T23:18:01.394+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): test.dbt_btc_transform.stg_kaggle__btcusdt> on 2024-10-15 23:12:44.688778+00:00
[2024-10-15T23:18:01.397+0000] {standard_task_runner.py:57} INFO - Started process 1053 to run task
[2024-10-15T23:18:01.401+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'run_dbt_models', 'test.dbt_btc_transform.stg_kaggle__btcusdt', 'manual__2024-10-15T23:12:44.688778+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/run_dbt_models.py', '--cfg-path', '/tmp/tmpencurant']
[2024-10-15T23:18:01.401+0000] {standard_task_runner.py:85} INFO - Job 50: Subtask test.dbt_btc_transform.stg_kaggle__btcusdt
[2024-10-15T23:18:01.402+0000] {cli_action_loggers.py:65} DEBUG - Calling callbacks: [<function default_action_log at 0x7f7b0c199c10>]
[2024-10-15T23:18:01.454+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:18:01.462+0000] {task_command.py:410} INFO - Running <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [running]> on host d867809303f2
[2024-10-15T23:18:01.462+0000] {settings.py:353} DEBUG - Disposing DB connection pool (PID 1053)
[2024-10-15T23:18:01.463+0000] {settings.py:212} DEBUG - Setting up DB connection pool (PID 1053)
[2024-10-15T23:18:01.463+0000] {settings.py:285} DEBUG - settings.prepare_engine_args(): Using NullPool
[2024-10-15T23:18:01.465+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [running]> from DB
[2024-10-15T23:18:01.507+0000] {taskinstance.py:1037} DEBUG - previous_execution_date was called
[2024-10-15T23:18:01.515+0000] {taskinstance.py:868} DEBUG - Clearing XCom data
[2024-10-15T23:18:01.547+0000] {retries.py:80} DEBUG - Running RenderedTaskInstanceFields._do_delete_old_records with retries. Try 1 of 3
[2024-10-15T23:18:01.562+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T23:12:44.688778+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T23:12:44.688778+00:00'
[2024-10-15T23:18:01.562+0000] {__init__.py:117} DEBUG - Preparing lineage inlets and outlets
[2024-10-15T23:18:01.563+0000] {__init__.py:155} DEBUG - inlets: [], outlets: []
[2024-10-15T23:18:01.564+0000] {bash.py:186} DEBUG - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='run_dbt_models' AIRFLOW_CTX_TASK_ID='test.dbt_btc_transform.stg_kaggle__btcusdt' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T23:12:44.688778+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T23:12:44.688778+00:00'
[2024-10-15T23:18:01.564+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-15T23:18:01.565+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            export DBT_SCHEMA=analytics &&\n            cd /opt/airflow/dbt &&\n            /home/airflow/.local/bin/dbt --no-write-json test --target dev --models stg_kaggle__btcusdt\n            ']
[2024-10-15T23:18:01.573+0000] {subprocess.py:86} INFO - Output:
[2024-10-15T23:18:03.692+0000] {subprocess.py:93} INFO - [0m23:18:03  Running with dbt=1.8.7
[2024-10-15T23:18:03.987+0000] {subprocess.py:93} INFO - [0m23:18:03  Registered adapter: ***=1.8.2
[2024-10-15T23:18:04.311+0000] {subprocess.py:93} INFO - [0m23:18:04  Found 1 model, 1 source, 540 macros
[2024-10-15T23:18:04.314+0000] {subprocess.py:93} INFO - [0m23:18:04
[2024-10-15T23:18:04.315+0000] {subprocess.py:93} INFO - [0m23:18:04  Nothing to do. Try checking your model configs and model specification args
[2024-10-15T23:18:04.902+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-15T23:18:04.930+0000] {__init__.py:75} DEBUG - Lineage called with inlets: [], outlets: []
[2024-10-15T23:18:04.930+0000] {taskinstance.py:789} DEBUG - Refreshing TaskInstance <TaskInstance: run_dbt_models.test.dbt_btc_transform.stg_kaggle__btcusdt manual__2024-10-15T23:12:44.688778+00:00 [running]> from DB
[2024-10-15T23:18:04.944+0000] {taskinstance.py:1356} DEBUG - Clearing next_method and next_kwargs.
[2024-10-15T23:18:04.945+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=run_dbt_models, task_id=test.dbt_btc_transform.stg_kaggle__btcusdt, execution_date=20241015T231244, start_date=20241015T231801, end_date=20241015T231804
[2024-10-15T23:18:04.945+0000] {taskinstance.py:2305} DEBUG - Task Duration set to 3.573594
[2024-10-15T23:18:04.959+0000] {cli_action_loggers.py:83} DEBUG - Calling callbacks: []
[2024-10-15T23:18:04.992+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-10-15T23:18:05.052+0000] {dagrun.py:697} DEBUG - number of tis tasks for <DagRun run_dbt_models @ 2024-10-15 23:12:44.688778+00:00: manual__2024-10-15T23:12:44.688778+00:00, state:running, queued_at: 2024-10-15 23:12:44.695918+00:00. externally triggered: True>: 3 task(s)
[2024-10-15T23:18:05.052+0000] {dagrun.py:718} DEBUG - number of scheduleable tasks for <DagRun run_dbt_models @ 2024-10-15 23:12:44.688778+00:00: manual__2024-10-15T23:12:44.688778+00:00, state:running, queued_at: 2024-10-15 23:12:44.695918+00:00. externally triggered: True>: 1 task(s)
[2024-10-15T23:18:05.054+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:12:44.688778+00:00 [None]> dependency 'Previous Dagrun State' PASSED: True, This task instance was the first task instance for its task.
[2024-10-15T23:18:05.055+0000] {taskinstance.py:1112} DEBUG - <TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:12:44.688778+00:00 [None]> dependency 'Not In Retry Period' PASSED: True, The task instance was not marked for retrying.
[2024-10-15T23:18:05.055+0000] {taskinstance.py:1103} DEBUG - Dependencies all met for dep_context=None ti=<TaskInstance: run_dbt_models.generate_dbt_docs manual__2024-10-15T23:12:44.688778+00:00 [None]>
[2024-10-15T23:18:05.058+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
